src/pandoc-export.ts

```diff
@@ -0,0 +1,109 @@
+'use strict';
+import * as vscode from 'vscode';
+import * as util from 'util';
+import * as child_process from 'child_process';
+import * as fs from 'fs';
+import * as path from 'path';
+
+const execSync = child_process.execSync;
+
+const userTemplateFilename = getUserTemplateFilename();
+let extensionTemplateFilename;
```

Comment:
why using a `let`?

Reply:
Because it's reassigned in line 33 and 84.

Reply:
I'll have a closer look then 🔍